src/pokemon_storage_system.c

```diff
@@ -9453,7 +9453,7 @@ u32 GetBoxMonLevelAt(u8 boxId, u8 boxPosition)
 #ifdef BUGFIX
     else
 #endif
-        lvl = 0;
+        lvl = 0; // NOLINT(readability-misleading-indentation)
```

Comment:
Just my opinion, but I kinda hate putting these sorts of "tool-only" comments in the code.

If you're committed to `clangd` not throwing a spurious warning here, is there a different way the code could be formatted which would read naturally and not throw the warning? Or maybe `clangd` could be told to assume `BUGFIX` is defined?

Reply:
Indenting the line with four spaces is fine for clang, if that's ok for you. I think clangd should have no warnings no matter the setting of BUGFIX.